src/routes/blog/posts/2021-04-14_edsa.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Was ist "EdDSA Support"?
+date: "2021-04-14"
+---
+
+das Signaturschema einfach erklärt!
+
+
+<!-- more -->
+
+Das aktuelle IOTA-Protokoll basiert auf dem Winternitz One-Time Signature (W-OTS) Schema, einem Hash-basierten Signaturschema, das die ternäre 243-Trit-Hash-Funktion Kerl verwendet. Dieses Signaturschema ist nachweislich resistent gegen einen ausreichend leistungsfähigen Quantencomputer, auf dem der Shor-Algorithmus läuft. Allerdings hat es im Gegensatz zu herkömmlichen ECDSA- oder EdDSA-Signaturen auch erhebliche Nachteile.
+
+**Zustandsabhängigkeit:** W-OTS erlaubt nur einen einzigen sicheren Signiervorgang. Ab der zweiten Signatur sind so viele Informationen offengelegt, dass der private Schlüssel und damit das Geld auf dieser Adresse als unsicher gilt. Dies stellt ein ernsthaftes Sicherheitsrisiko dar, da das Signieren einer einzigen ungültigen Transaktion als ebenso gefährlich angesehen werden muss wie die Offenlegung des privaten Schlüssels selbst.
+
+**Größe:** Die erzeugten Signaturen sind recht groß. In IOTA werden 2187 bis 6561 Trytes oder 1300 bis 3900 Bytes (je nach gewählter Sicherheitsstufe) für die Signatur reserviert.
+
+**Geschwindigkeit:** Sie basiert auf der Hash-Funktion Keccak-384 und wurde entwickelt, um einen Kompromiss zwischen Größe und Geschwindigkeit zu erreichen. So muss die Hashing-Funktion in der Standardeinstellung 702 Mal ausgeführt werden, um eine Signatur zu validieren, was selbst auf leistungsfähiger Hardware zu erheblichem System-Overhead führen kann.
+
+
+Mit Chrysalis wird das neue gebräuchliche Signaturschema Ed25519 eingeführt, welches das alte W-OTS Schema vollständig ersetzt. Das Ed25519 ist ein modernes EdDSA-Signaturschema, das SHA-512 und Curve25519 verwendet, es reduziert drastisch die Transaktionsgröße und ermöglicht folglich eine deutliche Erhöhung der möglichen messages per second (mps). Ein weiterer Vorteil ist die Wiederverwendbarkeit von Adressen, wodurch sich die Benutzerfreundlichkeit deutlich erhöht und auch die Implementierung der IOTA Technologie in Apps, Börsen oder anderen Systemen erheblich vereinfacht. Ein Nachteil ist allerdings, dass es weniger quanten-robust ist, dieses Problem haben derzeit aber die meisten herkömmlichen Verschlüsselungssysteme. Derzeit wird bereits weltweit an geeigneten neuen Signaturschemen geforscht und sobald es eine praktikable Lösung gibt, kann diese von IOTA einfach übernommen werden.
```

Comment:
Mit CHRYSALIS wird das neue gebräuchliche Signaturschema Ed25519 eingeführt, welches das alte W-OTS Schema vollständig ersetzt. Das Ed25519 ist ein modernes EdDSA-Signaturschema, das SHA-512 und Curve25519 verwendet, es reduziert drastisch die Transaktionsgröße und ermöglicht folglich eine deutliche Erhöhung der möglichen messages per second (mps). Ein weiterer Vorteil ist die Wiederverwendbarkeit von Adressen, wodurch sich die Benutzerfreundlichkeit deutlich erhöht und auch die Implementierung der IOTA Technologie in Apps, Börsen oder anderen Systemen erheblich vereinfacht. Ein Nachteil ist allerdings, dass es weniger quanten-robust ist, dieses Problem haben derzeit aber die meisten herkömmlichen Verschlüsselungssysteme. Derzeit wird bereits weltweit an geeigneten neuen Signaturschemen geforscht und sobald es eine praktikable Lösung gibt, kann diese von IOTA einfach übernommen werden.
 Spenden
@vromvonbeyond vromvonbeyond 15 minutes ago